### PR TITLE
Merge develop to main

### PR DESCRIPTION
**Pull request by smill3r**
This pull request will update main with the latest working version of develop, which already includes Load balancing, SSL on the reverse-proxy and the docker compose configurations for dev and build.

### CODE REVIEW

File: api/Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Use an official Node.js runtime as a parent image
+FROM node:18
```
Comment (gabyguerrero) [inline api/Dockerfile.dev:2]: Looks good! Just curious about the node image here and in Dockerfile, what's the difference between using one or the other?

**Review by gabyguerrero — APPROVED** [review]

File: api/package.json
Comment (gabyguerrero) [inline api/package.json:1]: Maybe we should also specify node version here:
"engines": {
    "node": "18.x"
  }

File: api/Dockerfile.dev (hunk 1 above)
Comment (smill3r) [reply api/Dockerfile.dev:2]: The other one uses a lighter version of the image in order to have a reduced image size for the final build.

File: api/package.json
Comment (smill3r) [reply api/package.json:1]: You're right, let's do that on another pull request 👍

**Pull request merged**